src/backend/templates/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-45266465-2"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-45266465-2');
+    </script>
+    <meta charset="utf-8">
+    <title>The Cycle-GAN App</title>
+    <link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='styles/style.css') }}">
+    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <header>
+      <h1>The Cycle-GAN App</h1>
+    </header>
+    <div>
+      <left>
+        <section>
+          <img id='uploaded_image' height='256' width='256'>
+        </section>
+        <form id='upload_form' action='' enctype='multipart/form-data' method='post'>
+          <div>
+            <input id='file_input' type="file" name="file" accept="image/*" required="true">
+          </div>
+        </form>
+        <p><strong>Status: </strong></p><p id='status_text'>Please select image to transform.</p>
+      </left>
+      <right>
+        <section>
+            <img id='inferred_image' height='256' width='256'>
+        </section>
+      </right>
+    </div>
+    <script>
+        function handleErrors(response) {
+          if (!response.ok) {
+              throw Error(response.statusText);
+          }
+          return response;
+        }
+        // uploads an image file to the server, fetches the inferred image and displays it 
+        function upload(file) {
+          let uploaded_image = document.getElementById('uploaded_image');
+          let status_text = document.getElementById('status_text')
+          uploaded_image.src = URL.createObjectURL(file);
+          const url = '/';
+          const data = new FormData();
+          data.append('file', file);
+          fetch(url, { method: 'POST',body: data})
+          .then(handleErrors)
+          .then(response => {
+            if(!response.ok) { 
+              throw response; 
+            }
+            return response.blob();
+          })
+          .then(blob => {
+            let image = document.getElementById('inferred_image');
+            const objectUrl = URL.createObjectURL(blob);
+            image.src = objectUrl;
+            status_text.innerHTML = 'Here you go. Feel free to upload another image.'
+          })
+          .catch(function(error) {
+            console.log("error occurred");
+            console.log(error);
+            status_text.innerHTML = error;
```

Comment:
Do you really have to use .innerHTML? This is usually frowned upon for security concerns.